app/Console/Kernel.php

```diff
@@ -3,28 +3,31 @@
 use Illuminate\Console\Scheduling\Schedule;
 use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
 
-class Kernel extends ConsoleKernel {
+class Kernel extends ConsoleKernel
+{
+    /**
+     * The Artisan commands provided by your application.
+     *
+     * @var array
+     */
+    protected $commands = [
+        'Symposium\Console\Commands\syncJoindInEvents',
```

Comment:
Yah, gonna do it in a separate pull though. Was already lowercase prior to this PR.